src/coreclr/inc/corinfo.h

```diff
@@ -387,18 +387,18 @@ enum CorInfoHelpFunc
     CORINFO_HELP_ULMOD,
     CORINFO_HELP_LNG2DBL,               // Convert a signed int64 to a double
     CORINFO_HELP_ULNG2DBL,              // Convert a unsigned int64 to a double
-    CORINFO_HELP_DBL2INT,
+    CORINFO_HELP_DBL2INT,               // unused
```

Comment:
```suggestion
    CORINFO_HELP_DBL2INT,
```
Reduce conflicts with https://github.com/dotnet/runtime/pull/97529